olog/httpx_client.py

```diff
@@ -227,14 +446,36 @@ async def aput_property(self, property):
         # extra verification that everything worked correctly.
         property_from_server = res.json()
         property_cp = property.copy()
-        property_cp['attributes'] = sorted(property_cp['attributes'],
-                                           key=lambda d: d['name'])
-        if OrderedDict(property_cp) != property_from_server:
-            raise UncaughtServerError(f"No http error was raised but server \
-                                      doesn't successfully put property you \
-                                      want. Server puts {res.json()} while \
-                                      you are tring to put {property}.")
-        return res.json()
-
-    def put_property(self, property):
-        return asyncio.run(self.aput_property(property))
+        for e in property_from_server['attributes']:
+            e.pop('state')
+        property_from_server.pop('state')
+        # property_cp['attributes'] = sorted(property_cp['attributes'], key=lambda d: d['name'])
+        # Looks server doesn't return sorted attributes
+        for i in property_cp['attributes']:
+            if i['name'] in monitor_list:
+                i['value'] = str(i['value'])
```

Comment:
I think it's worth understanding the server's intent a little better here. @shroffk If we send the server an integer, does it always convert it to string and then store that?

Reply:
@ke-zhang-rd Given Kunal's comment, maybe we should raise `ValueError` if the user tries to write a property that is anything but a string.

Reply:
Sure, you mean typerror right?

Reply:
Yep, exactly.